### ISSUE TITLE
Can't open new tab using :new command

### ISSUE THREAD

**Issue by cdecompilador** [VSCodeVim/Vim#5808]
**Describe the bug**
When using `:new` command the new tab opens an `Untitled`

**To Reproduce**
Let's say we are in a folder with two files, selecting one of them just type `:new otherfilename` 

**Expected behavior**
Open the requested tab

**Screenshots(GIF)**

![2020-12-02 11-29-47_2](https://user-images.githubusercontent.com/36774000/100862711-caab6880-3493-11eb-9dee-e879cf44a1c3.gif)

**Environment:**

```json 
{
	"messages": [],
	"activationTimes": {
		"codeLoadingTime": 25,
		"activateCallTime": 1,
		"activateResolvedTime": 2,
		"activationReason": {
			"startup": true,
			"extensionId": {
				"value": "vscodevim.vim",
				"_lower": "vscodevim.vim"
			},
			"activationEvent": "*"
		}
	},
	"runtimeErrors": []
}
```
- Extension Version: `1.17.2`
- OS Version: `Windows_NT x64 10.0.19042`
- VSCode version: `1.51.1`

**Comment by J-Fields** [comment]
Yeah, this should work. There would necessarily be a limitation though: VSCode doesn't have the concept of named in-memory buffers, so `:new` would need to actually save the file to disk immediately.

**Comment by stevenguh** [comment]
This should be possible, seems like a similar to issue to https://github.com/VSCodeVim/Vim/issues/5288#issuecomment-702909339. The file opening related command should be refactor in some way so they are in one place #5158
